dev-requirements.txt

```diff
@@ -1,4 +1,6 @@
 git+https://github.com/dbt-labs/dbt-adapters.git@main
+git+https://github.com/dbt-labs/dbt-adapters.git@main#subdirectory=dbt-tests-adapter
```

Comment:
We should install the github version of packages that we maintain because of cross-repo development.